neo/SmartContract/StandardService.cs

```diff
@@ -86,6 +87,18 @@ public void Dispose()
             Disposables.Clear();
         }
 
+        public long GetPrice(uint hash)
+        {
+            prices.TryGetValue(hash, out long price);
+            return price;
+        }
+
+        protected void Register(string method, Func<ExecutionEngine, bool> handler, long price)
+        {
+            Register(method, handler);
+            prices.Add(method.ToInteropMethodHash(), price);
+        }
+
```

Comment:
Good idea! Registering with price! I'm doing some tests now.

Reply:
👍 we frequently forget to add prices for new syscalls, this will ensure we don't in the future!